osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -38,6 +39,13 @@ public SliderBall(Slider slider, DrawableSlider drawableSlider = null)
 
             Children = new[]
             {
+                InternalFollowCircle = new CircularContainer
```

Comment:
The name of this drawable is definitely misleading given what it does, how about `TrackingArea` or `FollowingArea`, with an in-line comment explaining why it's there. (and reference to osu-stable)